ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -524,18 +543,21 @@ define([
     setRadiusUsng: function() {
       var usng = this.get('usng')
       if (usng !== undefined) {
-        var result = converter.USNGtoLL(usng, true)
+        let result
+        try {
+          result = converter.USNGtoLL(usng, true)
+        } catch (err) {}
```

Comment:
❓ same as others

Reply:
https://github.com/codice/usng.js/issues/21 needs to be resolved before I can remove this try catch